src/Type.h

```diff
@@ -425,6 +425,18 @@ struct Type {
         return code() == Handle;
     }
 
+    // Returns true iff type is an integral type where overflow is defined.
+    HALIDE_ALWAYS_INLINE
+    bool can_overflow_int() const {
+        return is_int() && bits() <= 16;
+    }
+
+    // Returns true iff type does have a well defined overflow behavior.
```

Comment:
nit: well-defined because well applies to defined, not to behavior.

Reply:
fixed in 4f0a437